### PR TITLE
docs: add docs for skills API

### PR DESCRIPTION
**Pull request by zyga**
Signed-off-by: Zygmunt Krynicki <zygmunt.krynicki@canonical.com>

### CODE REVIEW

File: docs/rest.md [hunk 1]
```diff
@@ -693,3 +693,50 @@ prerequisite in the database.
 The response is a stream of assertions separated by double newlines.
 The X-Ubuntu-Assertions-Count header is set to the number of
 returned assertions, 0 or more.
+
+
```
Comment (stevenwilkin) [inline docs/rest.md]: Extra empty line

File: docs/rest.md [hunk 2]
```diff
@@ -693,3 +693,50 @@ prerequisite in the database.
...
+## /2.0/skills
```
Comment (stevenwilkin) [inline docs/rest.md]: Could do with an empty line here

File: docs/rest.md [hunk 3]
```diff
@@ -693,3 +693,50 @@ prerequisite in the database.
...
+Available actions are:
+
+- grant: grant the skill to the given skill slot.
+- revoke: revoke the given skill from the given skill slot.
+
+Sample input:
+
+```javascript
+{
+    “action”: “grant” | “revoke”,
```
Comment (stevenwilkin) [inline docs/rest.md]: This isn't valid JSON

File: docs/rest.md (hunk 2 above)
Comment (zyga) [inline docs/rest.md]: Fixed

File: docs/rest.md (hunk 1 above)
Comment (zyga) [inline docs/rest.md]: Fixed

File: docs/rest.md (hunk 3 above)
Comment (zyga) [inline docs/rest.md]: Fixed

**Comment by mvo5** [comment]
:+1:

**Pull request merged**